ggml-metal.metal

```diff
@@ -2840,7 +2817,8 @@ kernel void kernel_cpy_f32_f16(
     for (int64_t i00 = tpitg.x; i00 < ne00; i00 += ntg.x) {
         device const float * src = (device float *)((device char *) src0 + i03*nb03 + i02*nb02 + i01*nb01 + i00*nb00);
 
-        dst_data[i00] = src[0];
+        // TODO: is there a better way to handle -INFINITY?
+        dst_data[i00] = src[0] == -INFINITY ? -MAXHALF : src[0];
```

Comment:
Not sure I fully understand the issue, but when we cast the `KQ_mask` from F32 to F16 in this kernel, the F32 `-INFINIFTY` values are converted to some F16 value that when multiplied with the ALiBi slope results in garbage. Even if I force the slope to be `1.0h` it still produces garbage. I expected that it would still be `-INFINITY`, but it's not the case. Since there is no way to print these values in Metal, this is the solution that I found to work, but it feels a bit poor